doc/Options/z390_options.md

```diff
@@ -0,0 +1,129 @@
+TRACEA or TRACE(A) | A | NO | Turn on az390 assembly trace generation on TRA file and turn off CON.  TRACEA will display each BAL statement during each pass of the assembler.
+TRACEALL | MALE | NO | Turn on all trace options generating TRM, TRA, TRL, and TRE files and turn off CON.
+TRACEC or TRACE(C) | M | NO | Trace copy file code for TRACEM and TRACEP.  The default is NOTRACEC to eliminated redundant code that usually just contains global variable declarations.
+TRACEG or TRACE(G) | E | NO | Turn on ez390 emulator trace generation on TRE file with trace of GETMAIN/FREEMAIN FQE memory management control block changes and turn off CON.
+TRACEI or TRACE(I) | M | NO | TRACEI will trace each AINSERT showing source on the TRM trace file.
+TRACEL or TRACE(L) | L | NO | Turn on lz390 linker trace generation on TRL file and turn off CON.  TRACEL shows each CSECT, ENTRY, EXTRN and OBJ file being loaded.
+TRACEM or TRACE(M) | M | NO | Turn on mz390 macro processor trace generation on TRM file and turn off CON.  TRACEM displays each conditional macro statement executed plus stored values and AIF compare values.
+TRACEP or TRACE(P) | M | NO | Turn on mz390 macro processor pseudo code generation and execution trace on TRM file and turn off CON.  TRACEP shows each conditional macro pseudo operation performed for each conditional macro statement and the values of all variables.
+TRACEQ or TRACE(Q) | E | NO | Turn on ez390 emulator QSAM/BSAM DCB I/O trace generation on TRE file and turn off CON.
+TRACES or TRACE(S) | M | NO | Displace MLC source lines on console regardless of CON setting to help locate any loops, waits, or hangs in large macro process.  If MCALL is also on, then *MCALL and *MEXIT comments will also be displayed to show location with nested macros.
+TRACET or TRACE(T) | E | NO | Turn on ez390 emulator TGET/TPUT and TCP/IO trace generation on TRE file and turn off CON.
+TRACEV or TRACE(V) | E | NO | Turn on ez390 emulator VSAM ACB/RPL I/O trace generation on TRE file and turn off CON.
+TRAP | E | YES | Trap any unexpected J2SE program exceptions and generate emulator 0C5 exception which can be handled by SPIE/STAE exits if defined.  NOTRAP can be used with Eclipse source debugger to stop at J2SE statement causing exception.
+TS | MALE | NO | Generate JDBC compatible time-stamp on all TRACE and ERR file records for use in debugging time dependent issue.
+VCB | E | YES | Generate VSAM cache buffer for improving VSAM performance by storing most recently accessed records and indexes.
+XREF | A | YES | Cross reference symbols to source lines in PRN assembly listing.
+ZSTRMAC | M | YES | Support expansion of ZSTRMAC structured conditional macro code instructions during loading of MLC, MAC, and CPY files.   Note z390 macros in z390\mac directory now are using zstrmac for structured coding.  NOZSTRMAC can only be used with MVS, VSE, or other non-structured macro libraries.
```

Comment:
Line 126. Remove extra space between first and second sentences. "... files.   Note ..." -> "... files.  Note ...".

Reply:
Fixed 40c73aa